.github/ISSUE_TEMPLATE/03-other.yml

```diff
@@ -0,0 +1,16 @@
+name: Other issue
+description: Other issue (except security reports, please file those in private!)
```

Comment:
We could create `SECURITY.md` once we agree on all the policies.

Reply:
Definitely! There is a dedicated issue for this: https://github.com/tact-lang/tact/issues/423